getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -331,50 +334,56 @@ Save the following in sof-dev.conf.
   BUILD_NOCLEAN = 1
 
 
-For Fedora and derived distributions, make the following changes:
+For targets running Fedora and derived distributions, make the following changes:
 
 .. code-block:: perl
 
-  GRUB_MENU    = "title" of the kernel entry as displayed by: 'grubby --info=ALL'
+  # GRUB_MENU should be the title of the custom kernel entry you added,
+  # which will match LOCALVERSION ("test") if you followed the previous steps
+  # You can view all your kernel entries with `grubby --info=ALL`
+  GRUB_MENU    = ${LOCALVERSION}
   GRUB_REBOOT  = grub2-reboot
   REBOOT_TYPE  = grub2bls
-  POST_INSTALL = ssh  -o 'ProxyCommand none' $SSH_USER@$MACHINE sudo dracut --hostonly --force --kver ${LOCALVERSION}
+  POST_INSTALL = ssh $SSH_USER@$MACHINE sudo dracut --hostonly --force /boot/initramfs-${LOCALVERSION}.img $KERNEL_VERSION
 
-7. Build and test
+4. Build and test
 -----------------
 
 .. code-block:: bash
 
+   # This can take a while, so don't kill it if it appears to freeze
    ./ktest.pl sof-dev.conf
 
 If this does not work, make sure you have all the following files in the
 local directory:
 
 * ktest.pl
 * sof-dev-cat
-* sof-dev
+* linux
 * sof-dev-build
 * sof-dev.conf
 * sof-dev-defconfig
 
-Ktest will compile, install the new kernel, and reboot. Prompt
-detection only works with a UART, not over SSH, so you will have to
-``Control-C`` manually when the console is not enabled.
+Ktest will compile and install the new kernel, then reboot the target device. Check which kernel is booted by running ``uname -r`` on the target.
+
+.. note::
+
+   KTest expects a UART connection to verify that the boot was successful. If you do not have a UART connection you will get some errors at the end of the ``ktest.pl`` script's execution, but you can ignore them as long as the custom kernel was installed and booted on the target device.
```

Comment:
```suggestion
   KTest expects a UART connection to verify that the boot was successful. If you do not have a UART connection you will get some errors at the end of the ``ktest.pl`` script's execution. You can either ignore them or change the TEST_TYPE to `install`
```

Reply:
I found out that changing the `TEST_TYPE` to `install` results in it skipping the `grub-reboot` command, so the default kernel is booted instead of the freshly installed one. It's possible there's a workaround but Pierre suggested I leave it how it is.